go16.go

```diff
@@ -9,7 +8,7 @@ import (
 	"runtime"
 )
 
-var gorootSrcPkg = filepath.Join(runtime.GOROOT(), "src", "pkg")
```

Comment:
I think the "pkg" needs to be removed from here, I must have grabbed it from the wrong file.